src/loader.js

```diff
@@ -360,6 +363,27 @@
                     function requireErrorHandler(err) {
                         console.log("Loader 'create' Error:\n" + err.message);
                     }
+                    function retrofit_114_to_118(state){
+                        var replacement_version = '1.18.0';
```

Comment:
Should spit out a warning to the console that it has encountered an old persist and is upgrading it.